.github/workflows/push-tag-release.yml

```diff
@@ -1,69 +1,267 @@
-# Create public release on tag push, preserves previous releases.
-# Re-tags "<package>/stable" to point on tag
+# Create public release on tag push, removes previous releases.
+# Publishes libraries to PyPi, building Connect plugin for non-libraries.
 
 name: CI-TAG
 
-# Prevent concurrent runs of this workflow due to the pants cache being shared between them.
-concurrency: pants_ci
-
 on:
   push:
-    tags-ignore:
-      - '**0.0.0**'
-      - '**latest**'
-      - '**stable**'
-    branches: ["dummy"] # Disable CI - do not trigger
+    tags:
+      - "**"
```

Comment:
is there anyway to filter this more? for instance, we always use `v*` in other projects